src/scripts/main.js

```diff
@@ -1 +1,24 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const thumbs = document.getElementById('thumbs');
+
+function updateMainImage(src) {
+  largeImg.src = src;
+}
+
+thumbs.addEventListener('click', (e) => {
+  e.preventDefault();
```

Comment:
Calling `e.preventDefault()` here prevents the default action of the click event, which is necessary to stop the browser from navigating to the link's href. This is correct if you want to prevent navigation, but ensure this is the intended behavior.